src/Card/CardMedia.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const CardMedia = (props) => {
```

Comment:
Sounds like a good candidate for reusing the `basicClassCreator` helper.
You can see these 2 as an example: https://github.com/tleunen/react-mdl/blob/master/src/Card/index.js#L4-L5

You can keep the tests though